website/docs/docs/on-premises/releases/1.34.0.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 1.34.0
+---
+
+## Release date
+
+22 October 2024
+
+## Features
+
+N/A
+
+## Fixes
+
+* fixes an issue where deleted idp/saml users who somehow still has useridentities records in the db, facing issues logging in.
```

Comment:
Can we improve this message? It's not clear to me what problem is solved and how it's useful to on-prem customers

Reply:
In fact - is this change relevant for on-prem? I think it's SaaS only: https://smartbear.atlassian.net/browse/PACT-2602

If so, I'd remove it.